.packit.yaml

```diff
@@ -50,3 +50,20 @@ jobs:
       - fedora-rawhide-ppc64le
       - fedora-rawhide-s390x
       - fedora-rawhide-x86_64
+  - job: copr_build
```

Comment:
You can make this slightly less verbose using the *YAML merge key syntax* which Packit supports - https://packit.dev/docs/configuration/#packit-service-jobs

Reply:
Good point, done.